### DISCUSSION TITLE
✏️  Termo: Sandboxed [machine, environment]

### DISCUSSION THREAD

**Discussion by contraexemplo** [infuse-l10n-pt-br/Infuse#5] [Glossário]
### Definição
Máquina ou ambiente isolado e seguro para conduzir experimentos com arquivos potencialmente maliciosos (por exemplo, arquivos executáveis).

**Comment by contraexemplo** [comment]
@yamakat Você já encontrou esse termo nos módulos que você traduziu? Sinto que os termos "ambiente isolado" e "ambiente seguro", por si só, não são capazes de transmitir a ideia — até porque a ideia também é transmitir o quão versátil a máquina ou o ambiente é, incluindo uma capacidade de descartá-la e começar do zero se necessário.

**Reply by yamakat** [reply]
não exatamente este termo ("sandbox"), mas algo como "máquina dedicada" ou "ambiente isolado", acho que sim (e mantive desse modo). gosto da ideia de adaptarmos conforme contexto, e incluir no glossário a metáfora de "ambiente minimamente controlado para experimentações", ou algo assim.

gostei muito desse glossário de Kubernetes!

**Comment by celsobessa** [comment]
Eu acho que isolado ou fechado para sandboxed capturam perfeitamente o sentido. Me parece que o que dá sentido de versatilidade e espaço para experimentação é o contexto. Sandbox tanto pode ser a caixa de areia onde crianças brincam, como a caixa de areia onde o gato vai deixar a b... rincadeira dele.

Mas talvez possamos sair da metáfora caixa de areia e encontrar outro termo em português com as mesmas propriedades de sandbox? Eu já usei "parquinho", quintal e variações como "parquinho fechado/particular/privado" para falar de ambientes "sandboxed". Não lembro se já usei para máquinas.

**Reply by contraexemplo** [reply]
Essa expressão é usada lá no começo para falar sobre boas práticas em investigações, então talvez valha a pena usar a expressão "máquina isolada" no texto corrido e expandir em um lugar como o glossário. Estava olhando como outros projetos usando Hugo fazem glossários, e o [modelo do Kubernetes](https://kubernetes.io/docs/reference/glossary/) te dá um espaço interessante de classificação, busca, e expansão. Cada definição é um arquivo Markdown, e uma página compila tudo.

**Reply by celsobessa** [reply]
Vou fuçar o modelo do Kubernetes e comento mais depois.

**Reply by celsobessa** [reply]
> Eu já usei "parquinho", quintal e variações como "parquinho fechado/particular/privado" para falar de ambientes "sandboxed". Não lembro se já usei para máquinas.

Complementando. Eu usei estes termos em conversas, oficinas, etc., não em textos/traduções
